src/Tasks/Common/Resources/Strings.resx

```diff
@@ -851,4 +851,12 @@ You may need to build the project on another operating system or architecture, o
     <value>NETSDK1186: This project depends on Maui Essentials through a project or NuGet package reference, but doesn't declare that dependency explicitly. To build this project, you must set the UseMauiEssentials property to true (and install the Maui workload if necessary).</value>
     <comment>{StrBegin="NETSDK1186: "}</comment>
   </data>
+  <data name="PackageContainsIncorrectlyCasedLocale" xml:space="preserve">
+    <value>Package {0} {1} has a resource with the locale '{2}'. This locale has been normalized to the standard format '{3}' to prevent casing issues in the build. Consider notifying the package author about this casing issue.</value>
```

Comment:
```suggestion
    <value>NETSDK1187: Package {0} {1} has a resource with the locale '{2}'. This locale has been normalized to the standard format '{3}' to prevent casing issues in the build. Consider notifying the package author about this casing issue.</value>
```